stepheng323/Automart
# #166860209 hosted app on Heroku

#### What does this PR do?
Here, I prepared the entire app for hosting on Heroku, added a Postgres database to Heroku and refactor the code for production enviroment.
#### Description of Task to be completed?

- Hosted on Heroku with all the endpoints working and a root endpoint to notify users about the app

- With the following endpoints working

POST /auth/signup
POST /auth/signin
POST /car/
POST /order/
PATCH /order/<:order-id>/price
PATCH /car/<:car-id>/status
PATCH /car/<:car-id>/price
GET /car/<:car-id>/
GET /car?status=available
GET /car?status=available&min_price=​ XXXValue​ &max_price=​ XXXValue
DELETE /car/<:car_id>/
GET /car/

#### How should this be manually tested?

- Open your  browser and navigate to https://automartendpoints.herokuapp.com/

- You should hit the root endpoint, for example, you can then navigate to the signup endpoints by adding 
  /api/v1/auth/signup to the address given above

#### What are the relevant pivotal tracker stories?
#166860209
